index.html

```diff
@@ -35,5 +35,144 @@ <h1 class="bigtext1">I'm Alexandra <br> Glad to see you!</h1>
       <li><a href="http://facebook.com/bestbynature"><i class="fa-brands fa-facebook"></i></a></li>
     </ul>
   </section>
+<!-- Begining of the works section -->
+  <section class="works-container">
+    <div class="works">
+      <div class="card-content">
+        <img src="./images/works1_img.png" alt="First works image" class="img-class">
+        <div class="text-border">
+        <p class="card-title">Tonic</p>
+        <p class="canopy">
+          <span class="span1">CANOPY</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2">Backend</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2">Dev 2015</span></p>
+          <p class="card-text">A daily selection of privately personalised reads: no accounts or sign-ups required.</p>
+          <div class="btns-3">
+            <ul>
+              <li><button type="button" class="btn3">html</button></li>
+              <li><button type="button" class="btn3">css</button></li>
+              <li><button type="button" class="btn3">javascript</button></li>
+            </ul>
+          </div>
+          <div>
+            <button type="button" class="btn-1">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+    
+    <div class="works">
+      <div class="card-content">
+        <img src="./images/works2_img.png" alt="First works image" class="img-class">
+        <div class="text-border">
+        <p class="card-title">Multi-Post <br> Stories</p>
```

Comment:
- [ ] For good semantic tags and as required for the project, kindly make sure the project title (e.g., "Tonic", ">Multi-Post <br> Stories",...) use a header tag `hx`(h3, h4,...)